docs/application/web/api/toc.xml

```diff
@@ -1982,6 +1982,9 @@
 						<topic href="html/ui_fw_api/Wearable_UIComponents/wearable_toggleswitch.htm" label="ToggleSwitch"></topic>
 						<topic href="html/ui_fw_api/Wearable_UIComponents/wearable_viewSwitcher.htm" label="ViewSwitcher"></topic>
 						<topic href="html/ui_fw_api/Wearable_UIComponents/wearable_virtuallist.htm" label="VirtualList"></topic>
+						<topic href="html/ui_fw_api/Wearable_UIComponents/wearable_numberpicker.htm" label="NumberPicker"></topic>
```

Comment:
@dorlinski I think you missed this file to update.

Reply:
Fixed